.vscode/settings.json

```diff
@@ -16,6 +16,9 @@
     { "language": "typescriptreact", "autoFix": true }
   ],
   "editor.formatOnSave": true,
+  "[php]": {
```

Comment:
Can we keep this a user-defined setting?

Reply:
We could, but I don't really want anyone auto-formatting our PHP at this stage. It can introduce runtime errors when trying to properly guess typehints as we've seen.